src/test/java/org/apache/log4j/util/SunReflectFilter.java

```diff
@@ -52,6 +52,12 @@ public String filter(String in) {
     if (in.indexOf("at java.lang.reflect.") >= 0) {
       return null;
     }
+    if (in.indexOf("at java.base/jdk.internal.reflect") >= 0) {
+      return null;
+    }
+    if (in.indexOf("at java.base/java.lang.reflect") >= 0) {
+      return null;
+    }
```

Comment:
SunReflectFilter has been replaced by Log4jAndNothingElseFilter

Reply:
This change was still required to make the tests pass

Reply:
Ah, I'm wrong I just misjudged the output